doc/ExpertOptions.md

```diff
@@ -1555,3 +1556,84 @@ dx describe /builds/1.02/applets/hello --json --details | jq '.details | .wdlSou
 # Recompilation
 
 Any significant WDL workflow is compiled into multiple DNAnexus applets and workflows. Naively, any modification to the WDL source would necessitate recompilation of all the constituent objects, which is expensive. To optimize this use case, all generated platform objects are checksumed. If a dx:object has not changed, it is not recompiled, and the existing version can be used. The checksum covers the WDL source code, the DNAnexus runtime specification, and any other attributes. There are two exceptions: the project name, and the folder. This allows moving WDL workflows in the folder hierarchy without recompilation.
+
+# Publishing global workflows
+
+<!-- TODO supported version pending release w/ APPS-989 -->
+Publishing a dxCompiler WDL workflow as a global workflow is supported from dxCompiler >= `v2.8.0`
+and dxpy >= `v<supported version here>`
```

Comment:
Pending release of supported dxpy version, to go here. Otherwise, this is ready for early review. @emiloslavsky @jdidion